en/deploy/README.md

```diff
@@ -1,246 +1,304 @@
+As you learned, a website has to be located on a server. There are a lot of providers, but we will use one of the ones with the simplest deployment process: [PythonAnywhere](http://pythonanywhere.com/). PythonAnywhere is free for small applications that don't have too many visitors, it'll definitely be enough for you now.
 
-We will be following this tutorial: https://devcenter.heroku.com/articles/getting-started-with-django, but we pasted it here so it's easier for you.
+The other external service we'll be using is [Github](http://www.github.com), which is a code hosting service.  There are others out there, but almost all programmers have a Github account these days, and now so will you!
 
-## The `requirements.txt` file
+We'll use Github as a stepping stone to transport our code to and from PythonAnywhere.
 
-We need to create a `requirements.txt` file to tell Heroku what Python packages need to be installed on our server.
 
-But first, Heroku needs us to install a few packages. Go to your console with `virtualenv` activated and type this:
+# Git
 
-    (myvenv) $ pip install dj-database-url gunicorn whitenoise
+Git is a "version control system" used by a lot of programmers - software which tracks changes to files over time so that you can recall specific versions later.  A bit like "track changes" in Microsoft Word, but much more powerful.
 
-After the installation is finished, go to the `djangogirls` directory and run this command:
 
-    (myvenv) $ pip freeze > requirements.txt
+## Installing Git
 
-This will create a file called `requirements.txt` with a list of your installed packages (i.e. Python libraries that you are using, for example Django :)).
+### Windows
 
-> __Note__: `pip freeze` outputs a list of all the Python libraries installed in your virtualenv, and the `>` takes the output of `pip freeze` and puts it into a file. Try running `pip freeze` without the `> requirements.txt` to see what happens!
+You can download Git from [git-scm.com](http://git-scm.com/).  You can hit "next next next" on all steps except for one; in the 5th step entitled "Adjusting your PATH environment", choose "Run Git and associated Unix tools from the Windows command-line" (the bottom option).  Other than that, the defaults are fine.  Checkout Windows-style, commit Unix-style line endings is good.
 
-Open this file and add the following line at the bottom:
+### MacOS
 
-    psycopg2==2.5.4
+Download Git from [git-scm.com](http://git-scm.com/) and just follow the instructions.
 
-This line is needed for your application to work on Heroku.
 
+### Linux
 
-## Procfile
+You probably have this installed already, so just try:
 
-Another thing we need to create is a Procfile. This will let Heroku know which commands to run in order to start our website.
-Open up your code editor, create a file called `Procfile` in `djangogirls` directory and add this line:
+    sudo apt-get install git  # on debian, ubuntu
```

Comment:
I'd rather not have the comments on the same lines as the commands. Might be too confusing.